doczrc.js

```diff
@@ -16,7 +16,7 @@ const BRAND = BRANDS[process.env.BRAND] ? process.env.BRAND : "bluehost";
 
 // doczrc.js
 export default {
-  title: `🐦 Bluebird - ${BRANDS[BRAND].name}`,
+  title: `🐦 Bluebird`,
```

Comment:
It seems Justhost styles are not compiling with JH variables.

Reply:
To be resolved in a future story.